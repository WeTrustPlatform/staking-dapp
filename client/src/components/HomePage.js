import React from 'react';
import Navbar from './Navbar';
import MainSection from './MainSection';
import FAQSection from './FAQSection';
import Footer from './Footer';
import ActivitiesSection from './ActivitiesSection';
import TopSection from './TopSection';

class HomePage extends React.Component {
  render() {
    return (
      <div>
        <Navbar />
        <TopSection color="light" />
        <MainSection color="dark" />
        <ActivitiesSection color="light" />
        <FAQSection color="dark" />
        <Footer color="light" />
      </div>
    );
  }
}

export default HomePage;
