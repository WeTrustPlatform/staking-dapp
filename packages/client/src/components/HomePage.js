import React from 'react';
import Navbar from './Navbar';
import MainSection from './MainSection';
import FAQSection from './FAQSection';
import Footer from './Footer';
import YourStakesSection from './YourStakesSection';
import TopSection from './TopSection';
import RankingSection from './RankingSection';
import WhyStakingSection from './WhyStakingSection';

class HomePage extends React.Component {
  render() {
    return (
      <div>
        <Navbar />
        <TopSection color="light" />
        <WhyStakingSection color="dark" />
        <MainSection color="light" />
        <YourStakesSection color="dark" />
        <RankingSection color="light" />
        <FAQSection color="dark" />
        <Footer color="light" />
      </div>
    );
  }
}

export default HomePage;
