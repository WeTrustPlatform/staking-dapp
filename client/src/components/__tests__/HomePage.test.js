import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import HomePage from '../HomePage';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<HomePage />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('tests snapshot', () => {
  const tree = renderer.create(<HomePage />).toJSON();
  expect(tree).toMatchSnapshot();
});
