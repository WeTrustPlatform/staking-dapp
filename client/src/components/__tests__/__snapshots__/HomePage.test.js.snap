// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests snapshot 1`] = `
<div>
  <div>
    <header
      className="MuiPaper-root-378 MuiPaper-elevation4-384 MuiAppBar-root-369 MuiAppBar-positionFixed-370 MuiAppBar-colorPrimary-376 mui-fixed"
    >
      <div
        className="MuiToolbar-root-405 MuiToolbar-regular-407 MuiToolbar-gutters-406"
      >
        <div
          className="Navbar-grow-367"
        >
          <img
            alt="WeTrust"
            className="Navbar-brandImage-368"
            src="wetrust-logo.svg"
          />
        </div>
        <ul
          className="MuiList-root-415 MuiList-padding-416"
        >
          <div
            className="Web3Account-accountContainer-413"
          >
            <div
              className="Web3Account-accountImageWrapper-414"
            >
              <img
                alt="metamask fox icon"
                height={21}
                src="meta-mask-logo.png"
                width={21}
              />
            </div>
            <div>
              <p
                className="MuiTypography-root-419 MuiTypography-body1-428 Web3Account-accountText-411"
              >
                Connect to Custom network
              </p>
            </div>
          </div>
        </ul>
      </div>
    </header>
  </div>
  <div
    className="Section-root-457 Section-light-459 TopSection-root-455"
    id="top-section"
  >
    <h4
      className="MuiTypography-root-419 MuiTypography-h4-434 MuiTypography-colorSecondary-450 MuiTypography-alignCenter-442 SectionHeader-container-460"
    >
      Help curate the SPRING directory
    </h4>
    <div
      className="MuiGrid-container-461 MuiGrid-justify-xs-center-478 TopSection-container-456"
    >
      <h6
        className="MuiTypography-root-419 MuiTypography-h6-436 MuiTypography-alignCenter-442"
      >
        As a decentralized donation platform, Spring allows Trustcoin token holders to decide which causes are highly ranked, and a way to add new causes to the platform
      </h6>
    </div>
  </div>
  <div
    className="Section-root-457 Section-dark-458"
  >
    <div
      className="MuiGrid-container-461 MuiGrid-justify-xs-center-478 WhyStakingSection-container-558"
    >
      <div
        className="MuiGrid-item-462 MuiGrid-grid-xs-12-501 MuiGrid-grid-sm-12-515 MuiGrid-grid-md-4-521 WhyStakingSection-gridItem-559"
      >
        <div>
          <img
            alt="increase-your-rank"
            src="icon-increase-rank.svg"
          />
        </div>
        <br />
        <h5
          className="MuiTypography-root-419 MuiTypography-h5-435"
        >
          Increase your Rank on the SPRING directory
        </h5>
        <br />
        <h6
          className="MuiTypography-root-419 MuiTypography-h6-436"
        >
          Stake your Trustcoin to help your favorite Cause more visible on SPRING. Causes with the most stake assigned to them will receive higher ranking on SPRING.
        </h6>
      </div>
      <div
        className="MuiGrid-item-462 MuiGrid-grid-xs-12-501 MuiGrid-grid-sm-12-515 MuiGrid-grid-md-4-521 WhyStakingSection-gridItem-559 WhyStakingSection-midItem-560"
      >
        <div>
          <img
            alt="buy-trst"
            src="icon-buy-trst.svg"
          />
        </div>
        <br />
        <h5
          className="MuiTypography-root-419 MuiTypography-h5-435"
        >
          How to get Trustcoin
        </h5>
        <br />
        <h6
          className="MuiTypography-root-419 MuiTypography-h6-436"
        >
          Please visit the following to obtain Trustcoin.
        </h6>
        <br />
        <div
          className="WhyStakingSection-row-561"
        >
          <div
            className="WhyStakingSection-rowItem-562"
          >
            <a
              href="https://www.bancor.network/"
              rel="noopener noreferrer"
              target="_blank"
            >
              <img
                alt="bancor"
                height="54"
                src="bancor.png"
                width="38"
              />
            </a>
          </div>
          <div
            className="WhyStakingSection-rowItem-562"
          >
            <a
              href="https://uniswap.io/"
              rel="noopener noreferrer"
              target="_blank"
            >
              <img
                alt="uniswap"
                height="42"
                src="uniswap.png"
                width="98"
              />
            </a>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-item-462 MuiGrid-grid-xs-12-501 MuiGrid-grid-sm-12-515 MuiGrid-grid-md-4-521 WhyStakingSection-gridItem-559"
      >
        <div>
          <img
            alt="nominate"
            src="icon-nominate.svg"
          />
        </div>
        <br />
        <h5
          className="MuiTypography-root-419 MuiTypography-h5-435"
        >
          Nominate any organization
        </h5>
        <br />
        <h6
          className="MuiTypography-root-419 MuiTypography-h6-436"
        >
          Stake your Trustcoin to nominate and reserve the top spot for your favorite Cause. Nominated Causes that have assigned Trustcoins will automatically receive preferential placement upon joining.
        </h6>
      </div>
    </div>
  </div>
  <div
    className="Section-root-457 Section-light-459"
    id="main-section"
  >
    <h4
      className="MuiTypography-root-419 MuiTypography-h4-434 MuiTypography-colorSecondary-450 MuiTypography-alignCenter-442 SectionHeader-container-460"
    >
      Start staking
    </h4>
    <div
      className="StakeNow-root-563"
    >
      <div
        aria-expanded={false}
        aria-haspopup="listbox"
        aria-labelledby="downshift-1-label"
        aria-owns={null}
        className="SearchInput-container-568"
        role="combobox"
      >
        <div
          className="MuiFormControl-root-570 MuiFormControl-marginNormal-571 MuiFormControl-fullWidth-573"
        >
          <div
            aria-activedescendant={null}
            aria-autocomplete="list"
            aria-controls={null}
            aria-labelledby="downshift-1-label"
            className="MuiInputBase-root-587 MuiInput-root-574 MuiInputBase-fullWidth-596 MuiInput-fullWidth-581 MuiInputBase-formControl-588 MuiInput-formControl-575"
            onClick={[Function]}
          >
            <input
              aria-invalid={false}
              autoComplete="off"
              className="MuiInputBase-input-597 MuiInput-input-582 MuiInputBase-inputType-600 MuiInput-inputType-585 MuiInputBase-inputTypeSearch-601 MuiInput-inputTypeSearch-586"
              disabled={false}
              id="downshift-1-input"
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              placeholder="Search cause name"
              required={false}
              type="search"
              value=""
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-container-461 MuiGrid-justify-xs-center-478 StakeNow-stake-564"
      >
        <div
          className="MuiGrid-item-462 MuiGrid-grid-xs-10-499 MuiGrid-grid-sm-8-511 MuiGrid-grid-md-4-521 StakeNow-button-567"
        >
          <button
            className="MuiButtonBase-root-630 MuiButton-root-604 MuiButton-contained-615 MuiButton-containedPrimary-616 MuiButton-raised-618 MuiButton-raisedPrimary-619 MuiButton-sizeLarge-628 MuiButton-fullWidth-629"
            disabled={false}
            onBlur={[Function]}
            onClick={[Function]}
            onContextMenu={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex="0"
            type="button"
          >
            <span
              className="MuiButton-label-605"
            >
              Stake Now
            </span>
            <span
              className="MuiTouchRipple-root-726"
            />
          </button>
        </div>
      </div>
    </div>
    <div
      className="MuiGrid-container-461 MuiGrid-justify-xs-center-478"
    >
      <h6
        className="MuiTypography-root-419 MuiTypography-h6-436"
      >
        <a
          className="HrefLink-link-652"
          href="https://www.bancor.network/"
          rel="noopener noreferrer"
          target="_blank"
        >
          Get more TRST
        </a>
      </h6>
    </div>
  </div>
  <div
    className="Section-root-457 Section-dark-458"
    id="activities-section"
  >
    <h4
      className="MuiTypography-root-419 MuiTypography-h4-434 MuiTypography-colorSecondary-450 MuiTypography-alignCenter-442 SectionHeader-container-460"
    >
      Your Stakes
    </h4>
    <div
      className="MuiPaper-root-378 MuiPaper-elevation2-382 MuiPaper-rounded-379 YourStakesSection-paper-653"
    >
      <table
        className="MuiTable-root-659 YourStakesSection-table-654"
      >
        <thead
          className="MuiTableHead-root-660"
        >
          <tr
            className="MuiTableRow-root-661 MuiTableRow-head-664"
          >
            <th
              aria-sort={null}
              className="MuiTableCell-root-666 MuiTableCell-head-667 MuiTableCell-paddingDense-671"
              scope="col"
            >
              Cause name
            </th>
            <th
              aria-sort={null}
              className="MuiTableCell-root-666 MuiTableCell-head-667 MuiTableCell-paddingDense-671"
              scope="col"
            >
              Stake amount
            </th>
            <th
              aria-sort={null}
              className="MuiTableCell-root-666 MuiTableCell-head-667 MuiTableCell-paddingDense-671"
              scope="col"
            >
              Locked until
            </th>
            <th
              aria-sort={null}
              className="MuiTableCell-root-666 MuiTableCell-head-667 MuiTableCell-paddingDense-671"
              scope="col"
            >
              Status
            </th>
            <th
              aria-sort={null}
              className="MuiTableCell-root-666 MuiTableCell-head-667 MuiTableCell-paddingDense-671"
              scope="col"
            >
              Tx hash
            </th>
          </tr>
        </thead>
        <tbody
          className="MuiTableBody-root-678"
        >
          <tr
            className="MuiTableRow-root-661"
          >
            <td
              aria-sort={null}
              className="MuiTableCell-root-666 MuiTableCell-body-668 MuiTableCell-paddingDense-671"
              colSpan={5}
            >
              <div
                className="YourStakesSection-noActivities-657"
              >
                You have 0 transaction in Your Stakes history. 
                <a
                  href="/#main-section"
                  onClick={[Function]}
                >
                  Please stake!
                </a>
              </div>
            </td>
          </tr>
        </tbody>
      </table>
    </div>
  </div>
  <div
    className="Section-root-457 Section-light-459"
  >
    <div
      className="MuiGrid-container-461 MuiGrid-justify-xs-center-478 RankingSection-container-679"
    >
      <div
        className="MuiGrid-item-462 MuiGrid-grid-xs-12-501 MuiGrid-grid-sm-12-515 MuiGrid-grid-md-6-523 RankingSection-gridItem-680"
      >
        <div>
          <div>
            <h5
              className="MuiTypography-root-419 MuiTypography-h5-435"
            >
              Most staked Causes
            </h5>
          </div>
          <div
            className="LeaderBoard-subtitle-682"
          >
            <h6
              className="MuiTypography-root-419 MuiTypography-h6-436"
            >
              The following causes have been granted higher ranking by Trustcoin holders.
            </h6>
          </div>
          <div
            className="LeaderBoard-causesStats-683"
          >
            <div
              className="MuiPaper-root-378 MuiPaper-elevation2-382 MuiPaper-rounded-379"
            >
              <table
                className="MuiTable-root-659"
              >
                <thead
                  className="MuiTableHead-root-660"
                >
                  <tr
                    className="MuiTableRow-root-661 MuiTableRow-head-664"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root-666 MuiTableCell-head-667 MuiTableCell-paddingNone-673"
                      scope="col"
                    >
                      Rank
                    </th>
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root-666 MuiTableCell-head-667 MuiTableCell-paddingNone-673"
                      scope="col"
                    >
                      Cause name
                    </th>
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root-666 MuiTableCell-head-667 MuiTableCell-paddingNone-673"
                      scope="col"
                    >
                      Staked amount
                    </th>
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root-666 MuiTableCell-head-667 MuiTableCell-paddingNone-673"
                      scope="col"
                    >
                      Stakers
                    </th>
                  </tr>
                </thead>
                <tbody
                  className="MuiTableBody-root-678 LeaderBoard-tableBody-687"
                >
                  <tr
                    className="MuiTableRow-root-661"
                  >
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root-666 MuiTableCell-body-668 MuiTableCell-paddingNone-673"
                      colSpan={4}
                    >
                      <div
                        className="LeaderBoard-loading-686"
                      >
                        <div
                          className="MuiCircularProgress-root-689 MuiCircularProgress-colorSecondary-693 MuiCircularProgress-indeterminate-691"
                          role="progressbar"
                          style={
                            Object {
                              "height": 40,
                              "width": 40,
                            }
                          }
                        >
                          <svg
                            className="MuiCircularProgress-svg-694"
                            viewBox="22 22 44 44"
                          >
                            <circle
                              className="MuiCircularProgress-circle-695 MuiCircularProgress-circleIndeterminate-697"
                              cx={44}
                              cy={44}
                              fill="none"
                              r={19}
                              strokeWidth={6}
                              style={Object {}}
                            />
                          </svg>
                        </div>
                      </div>
                    </td>
                  </tr>
                </tbody>
              </table>
            </div>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-item-462 MuiGrid-grid-xs-12-501 MuiGrid-grid-sm-12-515 MuiGrid-grid-md-6-523 RankingSection-gridItem-680 RankingSection-bottomItem-681"
      >
        <div>
          <div>
            <h5
              className="MuiTypography-root-419 MuiTypography-h5-435"
            >
              Nominated Causes
            </h5>
          </div>
          <div
            className="LeaderBoard-subtitle-682"
          >
            <h6
              className="MuiTypography-root-419 MuiTypography-h6-436"
            >
              The following causes have been nominated by Trustcoin holders.
            </h6>
          </div>
          <div
            className="LeaderBoard-causesStats-683"
          >
            <div
              className="MuiPaper-root-378 MuiPaper-elevation2-382 MuiPaper-rounded-379"
            >
              <table
                className="MuiTable-root-659"
              >
                <thead
                  className="MuiTableHead-root-660"
                >
                  <tr
                    className="MuiTableRow-root-661 MuiTableRow-head-664"
                  >
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root-666 MuiTableCell-head-667 MuiTableCell-paddingNone-673"
                      scope="col"
                    >
                      Rank
                    </th>
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root-666 MuiTableCell-head-667 MuiTableCell-paddingNone-673"
                      scope="col"
                    >
                      Cause name
                    </th>
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root-666 MuiTableCell-head-667 MuiTableCell-paddingNone-673"
                      scope="col"
                    >
                      Staked amount
                    </th>
                    <th
                      aria-sort={null}
                      className="MuiTableCell-root-666 MuiTableCell-head-667 MuiTableCell-paddingNone-673"
                      scope="col"
                    >
                      Stakers
                    </th>
                  </tr>
                </thead>
                <tbody
                  className="MuiTableBody-root-678 LeaderBoard-tableBody-687"
                >
                  <tr
                    className="MuiTableRow-root-661"
                  >
                    <td
                      aria-sort={null}
                      className="MuiTableCell-root-666 MuiTableCell-body-668 MuiTableCell-paddingNone-673"
                      colSpan={4}
                    >
                      <div
                        className="LeaderBoard-loading-686"
                      >
                        <div
                          className="MuiCircularProgress-root-689 MuiCircularProgress-colorSecondary-693 MuiCircularProgress-indeterminate-691"
                          role="progressbar"
                          style={
                            Object {
                              "height": 40,
                              "width": 40,
                            }
                          }
                        >
                          <svg
                            className="MuiCircularProgress-svg-694"
                            viewBox="22 22 44 44"
                          >
                            <circle
                              className="MuiCircularProgress-circle-695 MuiCircularProgress-circleIndeterminate-697"
                              cx={44}
                              cy={44}
                              fill="none"
                              r={19}
                              strokeWidth={6}
                              style={Object {}}
                            />
                          </svg>
                        </div>
                      </div>
                    </td>
                  </tr>
                </tbody>
              </table>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="Section-root-457 Section-dark-458"
    id="faq-section"
  >
    <h4
      className="MuiTypography-root-419 MuiTypography-h4-434 MuiTypography-colorSecondary-450 MuiTypography-alignCenter-442 SectionHeader-container-460"
    >
      FAQ
    </h4>
    <div
      className="FAQSection-container-699"
    >
      <div
        className="FAQItem-root-700"
      >
        <div
          className="FAQItem-question-703"
          onClick={[Function]}
          role="presentation"
        >
          <div
            className="MuiGrid-container-461"
          >
            <div
              className="MuiGrid-item-462 MuiGrid-grid-xs-10-499"
            >
              <h5
                className="MuiTypography-root-419 MuiTypography-h5-435"
              >
                General overview of Staking TRST
              </h5>
            </div>
            <div
              className="MuiGrid-item-462 MuiGrid-grid-xs-2-491 FAQItem-gridIcon-702"
            >
              <span
                aria-hidden="true"
                className="material-icons MuiIcon-root-705"
              >
                add
              </span>
            </div>
          </div>
        </div>
        <br />
        <hr
          className="MuiDivider-root-714 FAQItem-divider-701"
        />
      </div>
      <div
        className="FAQItem-root-700"
      >
        <div
          className="FAQItem-question-703"
          onClick={[Function]}
          role="presentation"
        >
          <div
            className="MuiGrid-container-461"
          >
            <div
              className="MuiGrid-item-462 MuiGrid-grid-xs-10-499"
            >
              <h5
                className="MuiTypography-root-419 MuiTypography-h5-435"
              >
                What does staking TRST token enable?
              </h5>
            </div>
            <div
              className="MuiGrid-item-462 MuiGrid-grid-xs-2-491 FAQItem-gridIcon-702"
            >
              <span
                aria-hidden="true"
                className="material-icons MuiIcon-root-705"
              >
                add
              </span>
            </div>
          </div>
        </div>
        <br />
        <hr
          className="MuiDivider-root-714 FAQItem-divider-701"
        />
      </div>
      <div
        className="FAQItem-root-700"
      >
        <div
          className="FAQItem-question-703"
          onClick={[Function]}
          role="presentation"
        >
          <div
            className="MuiGrid-container-461"
          >
            <div
              className="MuiGrid-item-462 MuiGrid-grid-xs-10-499"
            >
              <h5
                className="MuiTypography-root-419 MuiTypography-h5-435"
              >
                What is the benefit for staking TRST and why would I stake TRST?
              </h5>
            </div>
            <div
              className="MuiGrid-item-462 MuiGrid-grid-xs-2-491 FAQItem-gridIcon-702"
            >
              <span
                aria-hidden="true"
                className="material-icons MuiIcon-root-705"
              >
                add
              </span>
            </div>
          </div>
        </div>
        <br />
        <hr
          className="MuiDivider-root-714 FAQItem-divider-701"
        />
      </div>
      <div
        className="FAQItem-root-700"
      >
        <div
          className="FAQItem-question-703"
          onClick={[Function]}
          role="presentation"
        >
          <div
            className="MuiGrid-container-461"
          >
            <div
              className="MuiGrid-item-462 MuiGrid-grid-xs-10-499"
            >
              <h5
                className="MuiTypography-root-419 MuiTypography-h5-435"
              >
                How does my stake benefit the organization or the crypto ecosystem?
              </h5>
            </div>
            <div
              className="MuiGrid-item-462 MuiGrid-grid-xs-2-491 FAQItem-gridIcon-702"
            >
              <span
                aria-hidden="true"
                className="material-icons MuiIcon-root-705"
              >
                add
              </span>
            </div>
          </div>
        </div>
        <br />
        <hr
          className="MuiDivider-root-714 FAQItem-divider-701"
        />
      </div>
      <div
        className="FAQItem-root-700"
      >
        <div
          className="FAQItem-question-703"
          onClick={[Function]}
          role="presentation"
        >
          <div
            className="MuiGrid-container-461"
          >
            <div
              className="MuiGrid-item-462 MuiGrid-grid-xs-10-499"
            >
              <h5
                className="MuiTypography-root-419 MuiTypography-h5-435"
              >
                When can I unstake?
              </h5>
            </div>
            <div
              className="MuiGrid-item-462 MuiGrid-grid-xs-2-491 FAQItem-gridIcon-702"
            >
              <span
                aria-hidden="true"
                className="material-icons MuiIcon-root-705"
              >
                add
              </span>
            </div>
          </div>
        </div>
        <br />
        <hr
          className="MuiDivider-root-714 FAQItem-divider-701"
        />
      </div>
      <div
        className="FAQItem-root-700"
      >
        <div
          className="FAQItem-question-703"
          onClick={[Function]}
          role="presentation"
        >
          <div
            className="MuiGrid-container-461"
          >
            <div
              className="MuiGrid-item-462 MuiGrid-grid-xs-10-499"
            >
              <h5
                className="MuiTypography-root-419 MuiTypography-h5-435"
              >
                What's next after this?
              </h5>
            </div>
            <div
              className="MuiGrid-item-462 MuiGrid-grid-xs-2-491 FAQItem-gridIcon-702"
            >
              <span
                aria-hidden="true"
                className="material-icons MuiIcon-root-705"
              >
                add
              </span>
            </div>
          </div>
        </div>
        <br />
        <hr
          className="MuiDivider-root-714 FAQItem-divider-701"
        />
      </div>
      <div
        className="FAQItem-root-700"
      >
        <div
          className="FAQItem-question-703"
          onClick={[Function]}
          role="presentation"
        >
          <div
            className="MuiGrid-container-461"
          >
            <div
              className="MuiGrid-item-462 MuiGrid-grid-xs-10-499"
            >
              <h5
                className="MuiTypography-root-419 MuiTypography-h5-435"
              >
                What is TRST and where can I obtain TRST tokens so I can stake?
              </h5>
            </div>
            <div
              className="MuiGrid-item-462 MuiGrid-grid-xs-2-491 FAQItem-gridIcon-702"
            >
              <span
                aria-hidden="true"
                className="material-icons MuiIcon-root-705"
              >
                add
              </span>
            </div>
          </div>
        </div>
        <br />
        <hr
          className="MuiDivider-root-714 FAQItem-divider-701"
        />
      </div>
    </div>
  </div>
  <div
    className="Footer-container-721"
  >
    <div
      className="Footer-wrapper-722"
    >
      <a
        className="MuiTypography-root-419 MuiTypography-body1-428 Footer-link-720"
        href="/"
      >
        Home
      </a>
      <a
        className="MuiTypography-root-419 MuiTypography-body1-428 Footer-link-720"
        href="https://spring.wetrust.io/causes"
      >
        Causes
      </a>
      <a
        className="MuiTypography-root-419 MuiTypography-body1-428 Footer-link-720"
        href="https://www.wetrust.io"
      >
        About us
      </a>
      <a
        className="MuiTypography-root-419 MuiTypography-body1-428 Footer-link-720"
        href="/_Crypto_Unlocked_Terms_and_Conditions_v1.2.pdf"
      >
        Terms and Conditions
      </a>
      <a
        className="MuiTypography-root-419 MuiTypography-body1-428 Footer-link-720"
        href="https://github.com/WeTrustPlatform/documents/blob/master/FinclusionLabs_PrivacyPolicy_October92018_GDPRCompliant.pdf"
      >
        Privacy Policy
      </a>
    </div>
    <div
      className="Footer-wrapper-722"
    >
      <a
        className="MuiTypography-root-419 MuiTypography-body1-428 Footer-link-720 Footer-facebook-723"
        href="https://www.facebook.com/wetrustplatform"
      >
        Facebook
      </a>
      <a
        className="MuiTypography-root-419 MuiTypography-body1-428 Footer-link-720 Footer-twitter-724"
        href="https://twitter.com/wetrustplatform"
      >
        Twitter
      </a>
      <a
        className="MuiTypography-root-419 MuiTypography-body1-428 Footer-link-720 Footer-blog-725"
        href="https://blog.wetrust.io/"
      >
        Blog
      </a>
    </div>
    <div
      className="Footer-wrapper-722"
    >
      <p
        className="MuiTypography-root-419 MuiTypography-body1-428 Footer-copyright-719"
      >
        © 2019 WeTrustPlatform - All rights reserved
      </p>
    </div>
  </div>
</div>
`;
